pkg/cli/cli.go

```diff
@@ -93,6 +94,12 @@ func doMain(cmd *cobra.Command, cmdName string) error {
 
 	defer logcrash.RecoverAndReportPanic(context.Background(), &serverCfg.Settings.SV)
 
+	if cmd != nil {
+		// Handle --version. We do this here to ensure --version is
+		// supported on every sub-command.
```

Comment:
Supporting `--version` on every sub-command seems odd to me. Why would we want to do this? (even supporting `cockroach --version` seems unnecessary, but fine)

And where is the `--version` flag defined? is it a magic feature of command-line package?